ItemDescription3/tableEXT/salesLineTabExt.50401.TabExt.al

```diff
@@ -0,0 +1,12 @@
+tableextension 50401 appsalesLineTabExt extends "Sales Line"
+{
+    fields
+    {
+        field(50400; "appDescription 3"; Text[100])
+        {
+            DataClassification = CustomerContent;
+            TableRelation = Item."appDescription 3";
```

Comment:
@Ams272 why do you think Table Relation is required here? We don't want select descriptions from list of descriptions. Task tell us that we need transfer value from Item and that's it. Logically you can think it should be similar to standard Description or "Description 2"

Reply:
the only think i can think of is to use "flowfield and calcformula = lookup"

am i on the right track?

Reply:
As we discussed - no :) 
You don't need any flowfields here due to how other descriptions fields used in standard application (they are editable)

Reply:
i've reviewed the standard application. Kindly review my code if it is now correct